runtime

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# This file is meant to be sourced by the user, who can then run commands corresponding
+# to the name of the script to be run, e.g.
+#     runtime build                      (to run build.sh)
+#     runtime run                        (to run run.sh)
+#     runtime test integration/tc_68_2   (to run the automated test tc_68_2)
+
+# To source this file to make this function available, issue the command
+#     source runtime.sh
+
```

Comment:
Update these comments... how does it work now? Do you not have to put `./` in front of it?

Reply:
you don't have to add the ./ if you add it to your PATH. and I also made it callable from anywhere